CMakeLists.txt

```diff
@@ -7,14 +7,15 @@ set(CMAKE_DISABLE_SOURCE_CHANGES ON)
 set(CMAKE_DISABLE_IN_SOURCE_BUILD ON)
 
 ################
-# CGImap project
+# openstreetmap-cgimap project
 ################
 string(TIMESTAMP CURRENT_TIMESTAMP %y%m%d%H%M UTC)
 
-project(CGImap LANGUAGES CXX
-               VERSION "0.10.0.${CURRENT_TIMESTAMP}"
-               DESCRIPTION "CGImap is a C++ implementation of some parts of the OpenStreetMap API as a FastCGI process."
-               HOMEPAGE_URL "https://github.com/zerebubuth/openstreetmap-cgimap")
+project(openstreetmap-cgimap
```

Comment:
As a side note, this change is also visible in the generator header attribute returned by the endpoints:

generator="openstreetmap-cgimap 2.0.0 ..."

vs.

 generator="CGImap 0.9.3 ..."